Nennyfills/banka
# Ft swagger documentation 165711807

#### What does this PR do?
Publish swagger API documentation
#### Description of Task to be completed?
Add new endpoint to swagger documentation
Publish swagger documentation
#### How should this be manually tested?
[view](https://app.swaggerhub.com/apis-docs/Nennyfills/banka/1.0.0) check with required params or queries
#### What are the relevant pivotal tracker stories?
[#165711807](https://www.pivotaltracker.com/story/show/165711807)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/45515619/56942770-4608a380-6b14-11e9-8247-b4c3f411c5a3.png)
